src/rules-engine/data-processors/dgraph-data-processor.ts

```diff
@@ -119,6 +119,7 @@ export default class DgraphDataProcessor implements DataProcessor {
             data: {
               id: `${this.providerName}-provider`,
             },
+            rawData: {},
```

Comment:
Looks like these are load data for rule check, so I set the rawData to en empty object